### ISSUE TITLE
A11y_Azure Tools for VS Code [BM]_Subscriptions_Screen Reader: NVDA/Narrator is not reading name for "Select All Subscriptions" check box.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/vscode-azurefunctions#2678]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### **Actual Result:**
NVDA/Narrator is not reading name for "Select All Subscriptions" check box.

### **Expected Result:**
NVDA/Narrator should read "Select All Subscriptions" check box when focus lands on it.

### **GitHubTags:**
#Azure Tools for VS Code; #A11ySev1; #A11yTCS; #BM-Azure Tools for VS Code-Win-Feb2021; #AILimited; #DesktopApp; #Screen Reader; #WCAG1.3.1; #NVDA; #Win10; #Element:Checkbox; #Narrator;

### **OS Details:**
Windows 10 Version 20H2 (OS Build 19042.685)

### **Environment Details:**
Application Name: Azure Tools for VS Code
Version: 1.53.1
Commit: 5d424b828ada08e1eb9f95d6cb41120234ef57c7
Date: 2021-02-08T23:29:42.785Z
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19042
Azure Function Version: 1.2.1

### **Repro Steps:**

1. Open Visual studio code
2. Turn on NVDA/Narrator
3. Tab till extension button and press enter on it.
4. Search for "Azure Functions" and select it.
5. Tab till "Install" button and press enter on it.
6. Tab till "Azure" button from left side pane and press enter on it.
7. Default screen will get open.
8. Tab to Tree view and use down arrow to navigate to tree items.
9. Tab to "filter" and press enter on it.
10. Navigate to "Select All Subscriptions" and verify how NVDA/Narrator is reading it.

### **User Impact:**
Screen Reader users will not know the functionality of controls if screen reader is not reading the name of control complete information

### **Attachment for Reference:**
![2678_Narrator issue](https://user-images.githubusercontent.com/38523250/108308791-5f417f00-71d6-11eb-902f-a17b500dd033.png)

[2678_NVDA issue.zip](https://github.com/microsoft/vscode-azurefunctions/files/6019967/2678_NVDA.issue.zip)

**Comment by ejizba** [comment]
This should be filed directly on VS Code as I can repro with a simple quick pick like this:
```
await vscode.window.showQuickPick(['hello', 'world'], { canPickMany: true, ignoreFocusOut: true });
```
